src/ofxstatement/tool.py

```diff
@@ -149,7 +149,21 @@ def convert(args):
         statement = parser.parse()
     except exceptions.ParseError as e:
         log.error("Parse error on line %s: %s" % (e.lineno, e.message))
-        return 2  # error
+        return 2  # parse error
+    except exceptions.ValidationError as e:  # any assert_valid() may raise it
+        appui.warning("Problem with user input")
+        # log the exception with a traceback
+        log.exception(e)
```

Comment:
Will the user see the traceback in terminal? I'd really like to avoid that.

Reply:
Also, parser doesn't emit `ValidationError`, right? Or does it?

Reply:
Exactly, although the class accepts an object so it may be also a statement line or bank account. Should normally be raised in one of the assert_valid methods.